wd_core.au3

```diff
@@ -1105,7 +1105,7 @@ Func _WD_Option($sOption, $vValue = Default)
 
 		Case "console"
 			If $vValue == "" Then Return $_WD_CONSOLE
-			If Not (IsString($vValue) Or IsInt($vValue)) Then
+			If Not (IsString($vValue) Or IsInt($vValue) Or IsFunc($vValue)) Then
 				Return SetError(__WD_Error($sFuncName, $_WD_ERROR_InvalidDataType, "(string/int) $vValue: " & $vValue), 0, 0)

```

Comment:
Please update this message to reflect that "func" is a valid data type

Reply:
Done: [function/functionname](https://github.com/Danp2/au3WebDriver/pull/205/commits/8a0fc2a6bbf45bf9320756357861eeb54c8d280c)

Reply:
I don't understand why you added both function and functionname. Please explain.

Reply:
I must be missing something because I tried testing with the following --

```autoit
$x = "MsgBox"
ConsoleWrite('@@ Debug(' & @ScriptLineNumber & ') : IsFunc($x) = ' & IsFunc($x) & @CRLF & '>Error code: ' & @error & @CRLF) ;### Debug Console
```

The output was `IsFunc($x) = 0`, so how exactly is this supposed to work?

Reply:
My fault in understanding how `IsFunc()` works.
Fixed: [Function not function name](https://github.com/Danp2/au3WebDriver/pull/205/commits/b901669932f10bf7906326a5d36222859c9a9467)